rolling/release-rhel-build.yaml

```diff
@@ -37,6 +37,7 @@ package_blacklist:
 - octomap_server  # Build failures on RHEL
 - octovis  # Build failures on RHEL
 - ompl  # opende has no RPM package for RHEL
+- ros2_ouster_drivers # libtins-dev has no RPM package for RHEL
```

Comment:
I think these are supposed to be the package names. Did you mean `ros2_ouster`? 

https://github.com/ros/rosdistro/blob/a901084907b3f8c5cb7e65fb1631e1532ad7fb01/rolling/distribution.yaml#L3558

Reply:
Ah ok, I was even questioning that myself and in retrospect I chose poorly